pkg/detector/detector.go

```diff
@@ -276,6 +277,14 @@ func (d *ResourceDetector) EventFilter(obj interface{}) bool {
 			if found && secretType == string(corev1.SecretTypeServiceAccountToken) {
 				return false
 			}
```

Comment:
Can you add some comments to here?

Reply:
// The configmap and endpoints for leader election should be prevented from propagating. 

@chaunceyjiang